html-css/module-2/step3-instructions.md

```diff
@@ -0,0 +1,52 @@
+As we have seen, in order to override variables its important to understand how BEM is used to name them. It’s just as important when creating variables.
+
+## Workshop: Add a BEM element and custom styles with CSS custom property to support an icon element in the Label component
+
+a. We are going to add a custom icon element to the label component and apply a margin-right of 8px (--pf-global--spacer--sm)
+
+b. Let's assume there is a common use case for adding an icon to the left of text in the label component when used in your application. Start out by adding a label component.
+<span class="pf-c-label">
+    My custom label
+</span>
+
+c. Then place an icon before the label text
+<span class="pf-c-label">
```

Comment:
make sure you wrap these <spans> with ` too!

Reply:
done!!